examples/megacli.cpp

```diff
@@ -771,11 +841,7 @@ void DemoApp::printChatInformation(TextChat *chat)
     }
     if (!chat->title.empty())
     {
-        char *tstr = new char[chat->title.size() * 4 / 3 + 4];
-        Base64::btoa((const byte *)chat->title.data(), int(chat->title.size()), tstr);
-
-        cout << "\tTitle: " << tstr << endl;
-        delete [] tstr;
+        cout << "\tTitle: " << chat->title.c_str() << endl;
```

Comment:
The old code managed `chat->title` as a binary string.

Reply:
I've checked and the change is correct. The `TextChat::title` string stores the B64url encoded version of the corresponding array of bytes that represents the `ct`